packages/@react-spectrum/s2/src/Meter.tsx

```diff
@@ -82,11 +82,11 @@ const fillStyles = style<MeterStyleProps>({
   borderStyle: 'none',
   borderRadius: 'full',
   backgroundColor: {
-    default: 'informative',
+    default: 'informative-visual',
```

Comment:
matches what is in the design file. confirmed that the token `accent-content-color-default` which is listed in the token file is not correct with spectrum.